app/scripts/views/mixins/signin-mixin.js

```diff
@@ -24,13 +31,15 @@ define(function (require, exports, module) {
      * @return {object} promise
      */
     signIn: function (account, password) {
+      var self = this;
+      self.logEvent(`flow.${this.signInSubmitContext}.submit`);
```

Comment:
When will client side created flow events be used instead of server side created events? Up to this point, I thought all flow events were server side created.